test/functional/test-platform.js

```diff
@@ -210,9 +210,15 @@ describe('Platform', () => {
   });
 
   it('StandAlone', () => {
-    isStandAlone = true;
+    isStandalone = true;
     testStandAlone('Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_1 like Mac OS X)' +
         ' AppleWebKit/603.1.30 (KHTML, like Gecko) FxiOS/7.5b3349' +
-        ' Mobile/14E304 Safari/603.1.30', isStandAlone);
+        ' Mobile/14E304 Safari/603.1.30', isStandalone);
+  });
+
+  it('Not StandAlone', () => {
```

Comment:
Thinking about it now, instead of testing `isStandalone` by itself, we should test it for all of the above user agents. So instead of having `it('standalone', ...)` or `it('not standalone')` we should call `testStandalone` for each of the above `it('whatever browser')`s